inner/src/lib.rs

```diff
@@ -131,10 +159,25 @@ mod store {
         };
         // This is the unbounded default.
         match options.lru_max_entries {
-            None => (
-                quote::quote! { std::collections::HashMap<#key_type, #value_type> },
-                quote::quote! { std::collections::HashMap::new() },
-            ),
+            None => {
+				if let Some(hasher) = &options.custom_hasher {
```

Comment:
is there something going on with the indent here? Probably rustfmt will fix it :)

Reply:
Fixed in new commit